pkg/ttl/ttlworker/job_manager_integration_test.go

```diff
@@ -416,6 +394,10 @@ func TestTTLJobDisable(t *testing.T) {
 	store, dom := testkit.CreateMockStoreAndDomain(t)
 	tk := testkit.NewTestKit(t, store)
 
+	// turn off the `tidb_ttl_job_enable`
+	tk.MustExec("set global tidb_ttl_job_enable = 'OFF'")
```

Comment:
Set the `tidb_ttl_job_enable` to `OFF` before creating TTL table to make test stable